package.json

```diff
@@ -16,7 +16,8 @@
     "build": "node webpack/build.js",
     "test": "node --max-old-space-size=4096 node_modules/.bin/karma start",
     "test:watch": "node --max-old-space-size=4096 node_modules/.bin/karma start --no-single-run --auto-watch",
-    "lint": "eslint '{src,config}/**/*.{js,vue}' --cache --fix --max-warnings=0"
+    "lint": "eslint '{src,config}/**/*.{js,vue}' --cache --fix --max-warnings=0",
```

Comment:
I think now we need to use something like `lint:scripts` and `lint:css/styles` and one single `"lint": "lint:scripts && lint:css/styles"` in pre-{commit/push} hooks

Reply:
makes sense, could you add them please?

pls pay attention to `.ci/lint.sh` script that being executed automatically by the CI. Please make sure that the lint is still executable

I think we should pause merging of this PR since we got some active deleoped branches right now. Let’s merge it when next week, probably

Reply:
I will continue in a week, so we are still far from merging 🙂